visualization/app/codeCharta/ui/artificialIntelligence/artificialIntelligenceHelper/suspiciousMetricsHelper.ts

```diff
@@ -105,5 +113,8 @@ export function findGoodAndBadMetrics(
 		}
 	}
 
+	const untrackedMetrics = availableMapMetrics.filter(metric => !assessmentMetricList.includes(metric))
+	Array.prototype.push.apply(metricAssessmentResults.untrackedMetrics, untrackedMetrics)
```

Comment:
```suggestion
	metricAssessmentResults.untrackedMetrics.push(...untrackedMetrics)
```
Just a suggestion :)